src/lightkurve/interact.py

```diff
@@ -346,23 +404,53 @@ def add_gaia_figure_elements(tpf, fig, magnitude_limit=18):
 
     fig.add_tools(
         HoverTool(
-            tooltips=[
-                ("Gaia source", "@source"),
-                ("G", "@Gmag"),
-                ("Parallax (mas)", "@plx (~@one_over_plx{0,0} pc)"),
-                ("RA", "@ra{0,0.00000000}"),
-                ("DEC", "@dec{0,0.00000000}"),
-                ("pmRA", "@pmra{0,0.000} mas/yr"),
-                ("pmDE", "@pmde{0,0.000} mas/yr"),
-                ("x", "@x"),
-                ("y", "@y"),
-            ],
+            tooltips=tooltips,
             renderers=[r],
             mode="mouse",
             point_policy="snap_to_data",
         )
     )
-    return fig, r
+
+    # mark the target's position too
+    target_x, target_y = tpf.wcs.all_world2pix([[tpf.ra, tpf.dec]], 0)[0]
+    fig.cross(x=tpf.column + target_x, y=tpf.row + target_y, size=20, color="black", line_width=1)
+
+    # a widget that displays some of the selected star's metadata
+    # so that they can be copied (e.g., GAIA ID).
+    # It is a workaround, because bokeh's hover tooltip disappears as soon as the mouse is away from the star.
+    message_selected_target = Div(text="")
+
+    def show_target_info(attr, old, new):
+        if len(new) > 0:
+            msg = "Selected:<br><br>"
+            for idx in new:
+                tic_id = source.data['tic'].iat[idx]
```

Comment:
This line raised a KeyError when clicking on a target in a TPF obtained from TessCut, e.g. using
```python
lk.search_tesscut("Pi Men", sector=34).download(cutout_size=20).interact_sky()
```